.github/workflows/ci.yaml

```diff
@@ -124,10 +124,6 @@ jobs:
            github.event_name == 'push'
            && github.ref != 'refs/heads/master'
          )
-         || (
-           github.event_name == 'pull_request'
-           && !contains(github.event.pull_request.body, '[skip ci]')
-         )
```

Comment:
The conditional was short-circuiting here, there are tests for both [skip ci] and [skip test_windows_fast] below.